tests/sync/genesis/test_genesis_voting.py

```diff
@@ -0,0 +1,84 @@
+import time
+import pytest
+from pytest_testconfig import config as testconfig
+
+import tests.queries as q
+from tests.sync.test_sync import new_client_in_namespace
+from tests.test_bs import get_conf, setup_bootstrap, start_poet, add_curl
+from tests.utils import print_hits_entry_count
+
+# ==============================================================================
+#    TESTS
+# ==============================================================================
+
+
+# set a X nodes cluster
+# sleep in order to enable a block to be delivered
+# validate a block has been delivered
+# add an additional client while still in genesis
+# sleep until layer 4
+# validate no errors occurred
+# validate new node didn't receive any new blocks before being synced
+# @pytest.mark.xfail(strict=True, reason="new pod creates blocks before being synced")
+def test_unsync_while_genesis(init_session, setup_bootstrap, start_poet, add_curl):
+    time_before_first_block = 70
+    layers_to_wait = 4
+
+    layer_duration = int(testconfig['client']['args']['layer-duration-sec'])
+    bs_info = setup_bootstrap.pods[0]
+    cspec = get_conf(bs_info, testconfig['client'], None, setup_bootstrap.pods[0]['pod_ip'])
+
+    # Create a cluster of nodes
+    _ = new_client_in_namespace(testconfig['namespace'], setup_bootstrap, cspec, 9)
+
+    # Sleep to enable block creation
+    print(f"sleeping for {time_before_first_block} seconds in order to enable blocks to be published\n")
+    time.sleep(time_before_first_block)
+
+    # Validate a block was published
+    nodes_published_block, _ = q.get_blocks_per_node_and_layer(init_session)
+    if not nodes_published_block:
+        assert 0, f"no blocks were published during the first {time_before_first_block} seconds"
+
+    # Create a new node in cluster
+    unsynced_cl = new_client_in_namespace(testconfig['namespace'], setup_bootstrap, cspec, 1)
+
+    # Sleep until layers_to_wait layer, default is 4
+    print(f"sleeping for {layer_duration * layers_to_wait} seconds\n")
+    time.sleep(layer_duration * layers_to_wait)
+
+    # Found by Almogs: "validate votes failed" error has occurred following a known bug
```

Comment:
add a todo to remove this when bug is fixed

Reply:
I disagree, it's always nice to know this bug doesn't return